docker/test/integration/MiNiFi_integration_test_driver.py

```diff
@@ -141,7 +141,7 @@ def check_for_multiple_files_generated(self, file_count, timeout_seconds, expect
     def check_for_at_least_one_file_with_content_generated(self, content, timeout_seconds):
         output_validator = SingleOrMultiFileOutputValidator(decode_escaped_str(content))
         output_validator.set_output_dir(self.file_system_observer.get_output_dir())
-        self.check_output(timeout_seconds, output_validator, 1)
+        self.check_output(timeout_seconds, output_validator, timeout_seconds)
```

Comment:
This is actually intentional, because we want to allow more files to be generated and only check if at least one has the content we desire. The timeout seconds seemed to be a good limit as we do not generate more files than one every second.

Reply:
I see.  It would be better to make this clearer in the code, as currently it looks "wrong".  Something like this?
```suggestion
        expected_number_of_files = timeout_seconds
        self.check_output(timeout_seconds, output_validator, expected_number_of_files)
```

Reply:
I agree, that looks better, updated in 5ce4c54f1f26ab113d81b813c7f1a4370f7cc9a6